wgsl/index.bs

```diff
@@ -9389,217 +9389,6 @@ Issue: https://github.com/gpuweb/gpuweb/issues/1621
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>override</dfn> :
-
-    | `'override'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>pointer</dfn> :
-
-    | `'ptr'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>sampler</dfn> :
-
-    | `'sampler'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>sampler_comparison</dfn> :
-
-    | `'sampler_comparison'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>static_assert</dfn> :
-
-    | `'static_assert'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>struct</dfn> :
-
-    | `'struct'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_1d</dfn> :
-
-    | `'texture_1d'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_2d</dfn> :
-
-    | `'texture_2d'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_2d_array</dfn> :
-
-    | `'texture_2d_array'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_3d</dfn> :
-
-    | `'texture_3d'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_cube</dfn> :
-
-    | `'texture_cube'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_cube_array</dfn> :
-
-    | `'texture_cube_array'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_multisampled_2d</dfn> :
-
-    | `'texture_multisampled_2d'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_storage_1d</dfn> :
-
-    | `'texture_storage_1d'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_storage_2d</dfn> :
-
-    | `'texture_storage_2d'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_storage_2d_array</dfn> :
-
-    | `'texture_storage_2d_array'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_storage_3d</dfn> :
-
-    | `'texture_storage_3d'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_depth_2d</dfn> :
-
-    | `'texture_depth_2d'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_depth_2d_array</dfn> :
-
-    | `'texture_depth_2d_array'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_depth_cube</dfn> :
-
-    | `'texture_depth_cube'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_depth_cube_array</dfn> :
-
-    | `'texture_depth_cube_array'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>texture_depth_multisampled_2d</dfn> :
-
-    | `'texture_depth_multisampled_2d'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>uint32</dfn> :
-
-    | `'u32'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>vec2</dfn> :
-
-    | `'vec2'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>vec3</dfn> :
-
-    | `'vec3'`
-</div>
-<div class='syntax' noexport='true'>
-  <dfn for=syntax>vec4</dfn> :
-
-    | `'vec4'`
```

Comment:
This raises the question: If 'vec4' is just an identifier, what does it denote.   The Declaration and Scope section says an identifier will denote an object which is (a list of several things), but none of which is a thing that you can then add a type parameterization such as `<f32>`.  
And it's also not a context-dependent name, because `vec4` is used where an ordinary identifier can be used, depending on the tokens that follow.